src/Components/Analyzers/src/Resources.resx

```diff
@@ -180,4 +180,13 @@
   <data name="ComponentParametersShouldBeAutoProperties_Title" xml:space="preserve">
     <value>Component parameters should be auto properties</value>
   </data>
+  <data name="SupplyParameterFromFormShouldNotHavePropertyInitializer_Description" xml:space="preserve">
+    <value>Initializing a property decorated with [SupplyParameterFromForm] can be overwritten with null during a form post. To ensure the property is never null, move the initialization to a component lifecycle method.</value>
```

Comment:
Tweak the description.

The value of a property decorated with [SupplyParameterFromForm] and initialized with a property initialized can be overwritten with null when the component receives parameters.